charts/dapr/crds/configuration.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.2
+    controller-gen.kubebuilder.io/version: v0.11.3
```

Comment:
Do we have a version control for `kubebuilder`?

Reply:
I see it in Makefile, as we do not force developers use dev image to do it, it may be a hidden danger if `kubebuilder` breaks backward compatibility.